scripts/fetch-dashboard-assets.sh

```diff
@@ -2,14 +2,17 @@
 
 # This script is used to download built dashboard assets from the "GreptimeTeam/dashboard" repository.
 
-set -e
+set -x
 
 declare -r SCRIPT_DIR=$(cd $(dirname ${0}) >/dev/null 2>&1 && pwd)
 declare -r ROOT_DIR=$(dirname ${SCRIPT_DIR})
 declare -r STATIC_DIR="$ROOT_DIR/src/servers/dashboard"
+OUT_DIR="${1:-$SCRIPT_DIR}"
 
 RELEASE_VERSION="$(cat $STATIC_DIR/VERSION)"
 
+echo "Downloading assets to dir: $OUT_DIR"
+pushd $OUT_DIR
```

Comment:
You don't have to use `pushd`/`popd`, just `cd $OUT_DIR` is enough, but maybe check if `${OUT_DIR}` exists here.